### PR TITLE
OADP-4656: Update IBM P and Z support docs

### PR DESCRIPTION
**Pull request by anarnold97**
### JIRA

* [https://issues.redhat.com/browse/OADP-4656](https://issues.redhat.com/browse/OADP-4656)

### VERSION

* OCP 4.14+

### Link to docs preview:

* [OADP support for IBM Power and IBM Z](https://80012--ocpdocs-pr.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/oadp-features-plugins#oadp-support-for-ibm-power-and-ibm-z)

### QE review:
- [X] QE has approved this change.  - See [JIRA comment by IBM QE Ukthi Prasad](https://issues.redhat.com/browse/OADP-4699)

Additional information:

### CODE REVIEW

**Review by weshayutin — APPROVED** [review]
/LGTM

File: modules/oadp-ibm-power-test-support.adoc [hunk 1]
```diff
@@ -8,4 +8,5 @@
...
+* {ibm-power-name} running with {product-title} 4.12, 4.13, 4.14, and 4.15, and {oadp-short} {oadp-version-1-3} was tested successfully against an AWS S3 backup location target. Although the test involved only an AWS S3 target, Red Hat supports running {ibm-power-name} with {product-title} 4.13, 4.14, and 4.15, and {oadp-short} {oadp-version-1-3} against all S3 backup location targets, which are not AWS, as well.
+* {ibm-power-name} running with {product-title} 4.13, 4.14, 4.15, and 4.16, and {oadp-short} {oadp-version-1-4} was tested successfully against an AWS S3 backup location target. Although the test involved only an AWS S3 target, Red Hat supports running {ibm-power-name} with {product-title} 4.14, 4.15, and 4.16, and {oadp-short} {oadp-version-1-4} against all S3 backup location targets, which are not AWS, as well.
```
Comment (weshayutin) [inline modules/oadp-ibm-power-test-support.adoc:12]: sorry.. remove OCP 4.13 from OADP-1.4

**Review by weshayutin — CHANGES_REQUESTED** [review]
sorry.. remove OCP 4.13 from OADP-1.4

**Comment by weshayutin** [comment]
OADP 1.4 is supported on OCP 4.14+

**Review by weshayutin — CHANGES_REQUESTED** [review]
OADP 1.4 is supported on OCP 4.14+,  OCP 4.14, OCP 4.15, OCP 4.16

File: modules/oadp-ibm-power-test-support.adoc (hunk 1 above)
Comment (anarnold97) [inline modules/oadp-ibm-power-test-support.adoc:12]:
```suggestion
* {ibm-power-name} running with {product-title} 4.14, 4.15, and 4.16, and {oadp-short} {oadp-version-1-4} was tested successfully against an AWS S3 backup location target. Although the test involved only an AWS S3 target, Red Hat supports running {ibm-power-name} with {product-title} 4.14, 4.15, and 4.16, and {oadp-short} {oadp-version-1-4} against all S3 backup location targets, which are not AWS, as well.
```

File: backup_and_restore/application_backup_and_restore/oadp-features-plugins.adoc [hunk 2]
```diff
@@ -23,7 +23,9 @@ OpenShift API for Data Protection (OADP) is platform neutral. The information th
...
 * {oadp-short} 1.2.3 was tested successfully against {product-title} 4.12, 4.13, 4.14, and 4.15 for both {ibm-power-name} and {ibm-z-name}. The sections that follow give testing and support information for {oadp-short} 1.2.3 in terms of backup locations for these systems.
-* {oadp-short} 1.3.1 was tested successfully against {product-title} 4.13, 4.14, and 4.15 for both {ibm-power-name} and {ibm-z-name}. The sections that follow give testing and support information for OADP 1.3.1 in terms of backup locations for these systems.
+* {oadp-short} {oadp-version-1-3} was tested successfully against {product-title} 4.12, 4.13, 4.14, and 4.15 for both {ibm-power-name} and {ibm-z-name}. The sections that follow give testing and support information for {oadp-short} {oadp-version-1-3} in terms of backup locations for these systems.
+* {oadp-short} {oadp-version-1-4} was tested successfully against {product-title} 4.13, 4.14, 4.15, and 4.16 for both {ibm-power-name} and {ibm-z-name}. The sections that follow give testing and support information for {oadp-short} {oadp-version-1-4} in terms of backup locations for these systems.
```
Comment (anarnold97) [inline backup_and_restore/application_backup_and_restore/oadp-features-plugins.adoc:27]:
```suggestion
* {oadp-short} {oadp-version-1-4} was tested successfully against {product-title} 4.14, 4.15, and 4.16 for both {ibm-power-name} and {ibm-z-name}. The sections that follow give testing and support information for {oadp-short} {oadp-version-1-4} in terms of backup locations for these systems.
```

File: modules/oadp-ibm-z-test-support.adoc [hunk 3]
```diff
@@ -8,4 +8,5 @@
...
-* {ibm-z-name} running with {product-title} 4.13, 4.14, and 4.15, and {oadp-short} 1.3.1 was tested successfully against an AWS S3 backup location target. Although the test involved only an AWS S3 target, Red Hat supports running {ibm-z-name} with {product-title} 4.13 4.14, and 4.15, and {oadp-short} 1.3.1 against all S3 backup location targets, which are not AWS, as well.
+* {ibm-z-name} running with {product-title} 4.12, 4.13, 4.14, and 4.15, and {oadp-version-1-3} was tested successfully against an AWS S3 backup location target. Although the test involved only an AWS S3 target, Red Hat supports running {ibm-z-name} with {product-title} 4.13 4.14, and 4.15, and {oadp-version-1-3} against all S3 backup location targets, which are not AWS, as well.
+* {ibm-z-name} running with {product-title} 4.13, 4.14, 4.15, and 4.16, and {oadp-version-1-4} was tested successfully against an AWS S3 backup location target. Although the test involved only an AWS S3 target, Red Hat supports running {ibm-z-name} with {product-title} 4.14, 4.15, and 4.16, and {oadp-version-1-4} against all S3 backup location targets, which are not AWS, as well.
```
Comment (anarnold97) [inline modules/oadp-ibm-z-test-support.adoc:12]:
```suggestion
* {ibm-z-name} running with {product-title} 4.14, 4.15, and 4.16, and {oadp-version-1-4} was tested successfully against an AWS S3 backup location target. Although the test involved only an AWS S3 target, Red Hat supports running {ibm-z-name} with {product-title} 4.14, 4.15, and 4.16, and {oadp-version-1-4} against all S3 backup location targets, which are not AWS, as well.
```

**Review by weshayutin — APPROVED** [review]
/LGTM

**Review by weshayutin — APPROVED** [review]
/LGTM

**Review by weshayutin — APPROVED** [review]
/LGTM

File: _attributes/common-attributes.adoc [hunk 4]
```diff
@@ -39,6 +39,8 @@ endif::[]
 :oadp-full: OpenShift API for Data Protection
 :oadp-short: OADP
 :oadp-version: 1.4.0
+:oadp-version-1-3: 1.3.3
+:oadp-version-1-4: 1.4.0
```
Comment (sonia-garudi) [inline _attributes/common-attributes.adoc:43]: @anarnold97 the latest version validated for OADP 1.4 is 1.4.1. I suppose this should be set to `oadp-version-1-4: 1.4.1`. Please correct me if I'm wrong here. Thanks.

Comment (anarnold97) [inline _attributes/common-attributes.adoc:43]:
```suggestion
:oadp-version-1-4: 1.4.1
```

Comment (anarnold97) [reply _attributes/common-attributes.adoc]: @sonia-garudi - updated. if you are happy please add a LGTM

Thanks

**Comment by anarnold97** [comment]
PR approved by IBM QE in Jira - [OADP-4699](https://issues.redhat.com/browse/OADP-4699)

image

**Review by maxwelldb — APPROVED** [review]

**Comment by ShaunaDiaz** [comment]
@anarnold97 You can make manual cherry picks for the two failed ones, resolve any conflicts, and put those back in queue.

**Review by ShaunaDiaz — COMMENTED** [review]
/lgtm

**Pull request merged**